WORKSPACE

```diff
@@ -45,6 +45,47 @@ new_http_archive(
   sha256 = "3d374a730aef330424a356a8d4f04d8a54277c425e274ecb7d9c83aa912c6bfa"
 )
 
+# Dependencies to support TensorBoard functional tests.
+http_archive(
+    name = "io_bazel_rules_go",
+    sha256 = "ef1aa6a368808d3aa18cbe588924f15fb8fac75d80860080355595e75eb9a529",
+    strip_prefix = "rules_go-0.4.0",
+    url = "https://github.com/bazelbuild/rules_go/archive/0.4.0.tar.gz",
+)
+
+load("@io_bazel_rules_go//go:def.bzl", "go_repositories")
+
+go_repositories()
+
+http_archive(
+    name = "io_bazel_rules_webtesting",
+    sha256 = "6a5dd4c540eaa908754557f5baee54304f4a3b88de630d17d24c8b1767e7fda1",
+    strip_prefix = "rules_webtesting-bf1c94a3cc796ff5f8718009b5cac092459f3ce4",
+    url = "https://github.com/bazelbuild/rules_webtesting/archive/bf1c94a3cc796ff5f8718009b5cac092459f3ce4.tar.gz",
+)
+
+load("@io_bazel_rules_webtesting//web:repositories.bzl", "web_test_repositories")
+
+web_test_repositories(python = True)
+
+http_file(
+    name = "org_chromium_chromedriver_linux",
+    sha256 = "0c01b05276da98f203dc7eb4236c2ee7fe799b432734e088549bd0aadc71958e",
+    url = "http://chromedriver.storage.googleapis.com/2.24/chromedriver_linux64.zip",
+)
+
+http_file(
+    name = "fonts_noto_hinted_deb",
+    sha256 = "25b362c9437a7859ce034f22d94b698e8ed25007b443e5a26228ed5b3d2d32d4",
+    url = "http://bazel-mirror.storage.googleapis.com/http.us.debian.org/debian/pool/main/f/fonts-noto/fonts-noto-hinted_20160116-1_all.deb",
```

Comment:
Out of curiosity, is there a better place we can get these TTF files from? One thing I ended up noticing is that Debian deletes files from their server. The original URL for this is gone unfortunately. Unless some other Debian mirror exists that doesn't delete stuff.